### PR TITLE
Run gofmt only on Go 1.10

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
Signed-off-by: TomSweeneyRedHat <tsweeney@redhat.com>

I think this will work to run gofmt only when go is version 1.10.  I'd like to have this in travis.yml, but couldn't figure a clean way to do so.  Any other suggestions/comments gratefully accepted.

Once working, this should cure our go 1.11 CI issues.

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
This is failing with a syntax error, I've gone through a number of loops.  @rhatdan or @nalind any thoughts on what bit is out of place?

```
HEAD is now at 2c632d1a Merge pull request #1824 from cyphar/fix-mips-build-devNumber
ln -sf ../../opencontainers/runc/runc
# Run gofmt only on version 1.10
ifneq (,)
/bin/sh: 1: Syntax error: word unexpected (expecting ")")
make: *** [validate] Error 2
```

File: Makefile [hunk 1]
```diff
@@ -5,6 +5,7 @@ BINDIR := $(PREFIX)/bin
 BASHINSTALLDIR=${PREFIX}/share/bash-completion/completions
 BUILDFLAGS := -tags "$(AUTOTAGS) $(TAGS)"
 GO := go
+GOVERSION := go version
```
Comment (rhatdan) [inline Makefile]: GOVERSION := $(shell go version)

File: Makefile [hunk 2]
```diff
@@ -48,7 +49,10 @@ deps: gopath
 
 .PHONY: validate
 validate:
-	@./tests/validate/gofmt.sh
+	# Run gofmt only on version 1.10
+	ifneq (,$(findstring 1.10,$(GOVERSION)))
```
Comment (rhatdan) [inline Makefile]: We should only run this on 1.11?

Comment (TomSweeneyRedHat) [reply Makefile]: gofmt I thought was failing on 1.11, not 1.10, but regardless, I'm still getting a syntax error on this line.

File: Makefile (hunk 1 above)
Comment (TomSweeneyRedHat) [reply Makefile]: fixed, thx

File: Makefile (hunk 2 above)
Comment (rhatdan) [reply Makefile]: Can we fix our code to confirm with what golang 1.11 wants, or is golang 1.11 just broken?

**Comment by TomSweeneyRedHat** [comment]
@q384566678 tried to get the code happy for golang 1.11, but that ended up not working.  I'll try working from that angle too.

**Comment by TomSweeneyRedHat** [comment]
Ugh, I love make, the issue was the if statement was indented, it needed to be all the way to the left otherwise make treats it like a shell script.

**Comment by TomSweeneyRedHat** [comment]
@umohnani8 FYI

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**